ui/packages/catalog-ui-search/src/main/webapp/component/value/value.view.js

```diff
@@ -103,13 +102,6 @@ module.exports = Marionette.LayoutView.extend({
             })
           )
           break
-        case 'geometry':
```

Comment:
do you know when this `onBeforeShow` gets called? Was it just never hitting this case?